report/sections/Architecture/architecture.tex

```diff
@@ -2,7 +2,12 @@
 The platform allows lecturers to create a syllabus that includes problem sets, exercises and test cases for the exercises. Additionally, students can do the exercises that the lecturer has provided.
 The platform is a web application designed using a client-server architecture. Using this architecture the client communicates with a server using HTTP to send \textit{request} messages. The server waits for client requests and processes the requests when they arrive and replies to the requests with a \textit{response} message.
 
-The platform has a user interface as well as business logic to handle user requests. The platform supports role-based authentication with roles, meaning that users of varying roles have access to different functionality. The platform can run tests, to check whether a piece of code solves an exercise. Account data, test runs, and syllabus data is persisted in a database.
+The platform has a frontend component containing a user interface and logic to communicate to the backend component. 
```

Comment:
4 of the first 7 lines start with `the platform`, seems a bit excessive

Reply:
Det skal komme i andre rettelser, ikke dem fra carlos